quickget

```diff
@@ -945,7 +945,7 @@ EOF
         esac
 
         if [ "${OS}" == "ubuntu" ] && [[ ${RELEASE} == *"daily"*  ]]; then
-        # wont install  lobster testing with less than 18GB
+        # won't install lobster testing with less than 32GB
```

Comment:
he minimum for lobster testing is 18G
`quickget` adds some extra headroom but the comment documents the true minimum

Please tweak this change retaining just the typo corrections, optionally with a mention of the 32G allocation with headroom if you wish